README.md

```diff
@@ -94,12 +94,21 @@ git checkout -b your-feature-branch-name
 ```bash
 git add .
 git commit -m "Add your commit message here"
+``` 
+3. Running Test
```

Comment:
This would read more naturally with: `3. Running Tests`

I generally don't care to nitpick spelling/English like this but I think it's good to get it correct in a README.

Reply:
100% Good catch!